resources/views/event-submission-form/url.antlers.html

```diff
@@ -0,0 +1,29 @@
+<div class="tw-relative {{ wrapper_classes }}">
+    <span class="tw-absolute tw-left-3 tw-top-[calc(50%-8px)]">
+        {{ svg:chain class="tw-w-4 tw-h-4 tw-text-brand-grey" }}
+    </span>
+
+    <input
+        type="url"
+        name="{{ field_name }}"
+        id="{{ field_name }}"
+        {{ if old[field_name] }}
+            value="{{ old[field_name] }}"
+        {{ /if }}
+        placeholder=" "
+        class="tw-pl-9 tw-peer tw-bg-brand-grey/5 tw-px-3 tw-pt-[18px] tw-pb-2 tw-border-b-2 tw-border-brand-grey/80 motion-safe:tw-transition-colors tw-border-x-0 tw-border-t-0 focus-within:tw-border-brand-teal focus:tw-outline-none tw-text-base tw-text-brand-grey {{ class }}"
+    >
+    <label
+        class="tw-pointer-events-none tw-absolute motion-safe:tw-transition-all tw-text-brand-grey/80 tw-text-xs tw-top-1 tw-left-9 tw-font-medium
+            peer-placeholder-shown:tw-left-9 peer-placeholder-shown:tw-top-4 peer-placeholder-shown:tw-font-normal peer-placeholder-shown:tw-text-base
+            peer-focus:tw-text-xs peer-focus:tw-top-1 peer-focus:tw-font-medium
+        "
+        for="{{ field_name }}"
+    >
+        {{ field_label }}
+    </label>
+
+    {{ get_error:{field_name} }}
```

Comment:
New ulr validation errors aren't appearing for me

Reply:
Appears to be frontend html validation, although normally the browser would highlight it with a tooltip but that didn't happen for some reason. I think the datetime validation might be the same issue.